File: 1018-Banknotes/1018.py
money = int(input())
banknotes = [100,50,20,10,5,2,1]
print(money)
for i in range(len(banknotes)):
        notes = money // banknotes[i]
        money = (money % banknotes[i])
        print( "{} nota(s) de R$ {},00".format(notes,banknotes[i]))

File: 1018-Banknotes/1018.go
package main

import "fmt"

func main() {

	var money, notes int
	var banknotes = [7]int{100, 50, 20, 10, 5, 2, 1}

	fmt.Scan(&money)

	fmt.Printf("%d\n", money)
	for i := 0; i < len(banknotes); i++ {
		notes = int(money / banknotes[i])
		money = money % banknotes[i]
		fmt.Printf("%d nota(s) de R$ %d,00\n", notes, banknotes[i])
	}

}
